src/Perpetuum/Services/EventServices/EventProcessors/NpcStateAnnouncer.cs

```diff
@@ -3,6 +3,7 @@
 using Perpetuum.Services.Channels;
 using Perpetuum.Services.EventServices.EventMessages;
 using Perpetuum.Zones.NpcSystem.Flocks;
+using Perpetuum.Log.Loggers;
```

Comment:
Maybe some import used for debugging? Is it still needed here?

Reply:
Maybe you will want to keep this around depending on your thoughts on the approach alternatives below

Reply:
I'd intended to add something to log the missing values, but forgot to actually do it.